### ISSUE TITLE
Not server compatible at all.

### ISSUE THREAD

**Issue by Si1kn** [AlephInfinity1/ForgeBlock#10]
This mod can't be ran on servers because of some client side classes, starting fixing this now.

**Comment by Si1kn** [comment]
For example: Minecraft.class is client side only, and since the mod uses that way, the server freaks out since it cant find that client side class.

**Comment by AlephInfinity1** [comment]
Thanks, kinda knew this sided issue would trip me at some point, since I wasn't aware of this issue when I first started creating this mod. Thanks for the maintenance updates!

**Comment by Si1kn** [comment]
Hey no problem, looks like we need to have 2 diff projects, as theres alot of just client side code. Not sure yet but seems like it.

**Comment by Si1kn** [comment]
Also do you have discord @AlephInfinity1 ?

**Comment by AlephInfinity1** [comment]
Yep, my discord is Redpo888#9897 . Might not be very active during weekdays though since school WiFi blocks discord, sorry about that

**Comment by Deftu** [comment]
By the looks of it we might need to make a seperate jar for the server side and just allow it to crash the server or print something to console
